masterlist.yaml

```
@@ -17752,6 +17752,44 @@ plugins:
       - crc: 0xFBAF34B2
         util: 'TES4Edit v4.0.3i (RC1 for 4.0.4)'
 
+  - name: 'Eastbrink(Basic|CM)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/36619/'
+        name: 'Eastbrink Town'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Eastbrink Town ESP' ]
+        condition: 'many("Eastbrink(Basic|CM)\.esp")'
+    tag:
+      - C.MiscFlags
+      - C.Water
+  - name: 'EastbrinkBasic.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Bravil Sea Domes'
+          - '[Eastbrink Patches](https://www.nexusmods.com/oblivion/mods/45365/)'
+        condition: 'active("BravilSeaDomes.esp") and not file("Eastbrink-SeaDomes Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'ImpeREAL Empire - Unique Forts'
+          - '[Eastbrink Patches](https://www.nexusmods.com/oblivion/mods/45365/)'
+        condition: 'active("ImpeREAL Empire - Unique Forts.esp") and not active("(UL[MS] )?Eastbrink-ImpeREALForts(-SilverfishRiverValley)? Patch\.esp")'
+      - <<: *useVersion
```

The CM Partners version just makes a few NPCs recruitable followers, so it doesn't really warrant an error or warning:
```suggestion
      - <<: *useVersion
        type: say
```